code_and_tutorials/reactleaflet_fritekstsok_maptiles_matrikkelkart_example/src/components/ApiKey.tsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react'
 import { useRecoilState } from 'recoil';
 import { apiKey } from '../state/state'
-import { Box , TextField } from '@material-ui/core';
+import { Box , Link, TextField, Typography } from '@material-ui/core';
 
 interface Props {
     
 }
 
 export const ApiKey = (props: Props) => {
-    const [key, setKey]= useRecoilState(apiKey);
+    const [key, setKey]= useRecoilState<string | null>(apiKey);
     const handleTextfieldChange = (e: any) => {
-        if (e.target.value.length == 36) {
+        if (e.target.value.length === 36) {
             setKey(e.target.value);
         }
     }
 
     return (
-        key.length < 36 ?
+        !key || key.length < 36 ?
         <Box display="flex" flexDirection="column"justifyContent='center' alignItems='center' padding={10}>
+            <Typography variant="h4">Demo of Norkart Maptiles, Fritekstsøk and Matrikkelkart</Typography>
```

Comment:
Vi bør kanskje også si en setning om hva disse tjenestene er og linke til mer info? Kan være en task for neste sprint.

Reply:
Task laget.